src/data_provider/src/packages/pkgWrapper.h

```diff
@@ -41,7 +39,12 @@ class PKGWrapper final : public IPackageWrapper
             , m_vendor{UNKNOWN_VALUE}
             , m_installTime {UNKNOWN_VALUE}
         {
-            getPkgData(ctx.filePath + "/" + ctx.package + "/" + APP_INFO_PATH);
+            std::string pathInfoPlist = ctx.filePath + "/" + ctx.package + "/" + INFO_PLIST_PATH;
+
+            if (Utils::existsRegular(pathInfoPlist))
```

Comment:
Previous to this modification, the pkgWrapper didn't explicitly check for the existence of the file anywhere. The behavior may have been the same, but not because this effect is clearly wanted.
If the file didn't exist, the isBinary method call failed, and then the file attempt to open also failed. I think that an explicit validation of the file's existence is an improvement.

Reply:
It's an improvement, yes.
Thank you!